mongo/integration/sdam_prose_test.go

```diff
@@ -31,7 +31,8 @@ func TestSDAMProse(t *testing.T) {
 	heartbeatIntervalMtOpts := mtest.NewOptions().
 		ClientOptions(heartbeatIntervalClientOpts).
 		CreateCollection(false).
-		ClientType(mtest.Proxy)
+		ClientType(mtest.Proxy).
+		MinServerVersion("4.4") // RTT Monitor / Streaming protocol is not supported for versions < 4.4.
```

Comment:
Before this change, we always started the RTT monitor. The goals of this ticket is to avoid opening a connection for streaming for servers that do not support streaming: https://github.com/mongodb/specifications/pull/1452/files#diff-67f5deaee17cfd34b1845819d83009a9b7e16d2ef28ffce946f51b714474efdeR745